cmd/kubeadm/app/cmd/init.go

```diff
@@ -65,22 +65,27 @@ func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 		&s.InitFlags.API.ExternalDNSNames, "api-external-dns-names", []string{},
 		`(optional) The DNS names to advertise, in case you have configured them yourself.`,
 	)
-
 	cmd.PersistentFlags().IPNetVar(
 		&s.InitFlags.Services.CIDR, "service-cidr", *kubeadmapi.DefaultServicesCIDR,
-		`(optional) use alternative range of IP address for service VIPs, e.g. "10.16.0.0/12"`,
+		`(optional) Use alterantive range of IP address for service VIPs, defaults to `+
+			kubeadmapi.DefaultServicesCIDRString,
+	)
+	cmd.PersistentFlags().IPNetVar(
+		&s.InitFlags.PodNetwork.CIDR, "cluster-cidr", net.IPNet{},
+		`(optional) Use alternative range of IP addresses for the pod overlay network VIPs, e.g. 10.3.0.0/16.`+
```

Comment:
Sorry about the VIPs, it's a copy-paste residual. 
> e.g. in the where routes are setup in the cloud cloud provider.

I want to understand this correctly - by this you mean that the RouteController can still use the `clusterCIDR` to establish routes to existing cloud provider instances (separate from the cluster) so that the cluster can talk to them? Is that the use case? Or am I misinterpreting this part ... 

I just have not had this flag used for anything else but overlays in my experience, so it would be nice to get a better grasp on it.